examples/oauth2/auth-server/.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
+.dockerignore
+Gemfile.lock
```

Comment:
Sure you don't want this to lock the versions of the gems that your bundle install installs?